plugins/window_size/lib/src/window_size_channel.dart

```diff
@@ -56,6 +56,14 @@ const String _setWindowMinimumSizeMethod = 'setWindowMinimumSize';
 /// unconstrained in that dimension.
 const String _setWindowMaximumSizeMethod = 'setWindowMaximumSize';
 
+/// The method name to set the window title of a window.
+const String _setWindowTitleMethod = 'setWindowTitle';
+
+/// The method name to set the window title's represented URL.
+/// 
+/// On macOS, if the URL is a file URL, the window shows an icon in its title bar. 
```

Comment:
Does this concept even exist on other platforms?

Reply:
I suspect not.

I'd like this to enable putting together a text editor on macOS that ties into the capabilities of the system, but the user side code is going to need to platform sniff to be able to use this sanely.

Reply:
Let's say "Only implemented for macOS." instead then, until/unless we figure out if there's some way to make a more general-purpose API that would apply to other platforms.

Reply:
Done